api/controllers/UserController.js

```diff
@@ -117,10 +117,17 @@ module.exports = {
     try {
       // If the user is logged in, see if we can find them first to verify they exist
       const { username } = await userAPI.findUser(id);
+
+      // If the user is currently in a game, we need to populate the game
+      if (game) {
+        gameService.populateGame({ gameId: game });
+      }
```

Comment:
Doesn't this throw the value away? `gameService.populateGame` returns a value wrapped in a promise; it shouldn't have any side effects. What's this doing?

Reply:
This sorta speaks to my confusion about the type of `game` in the obj param above because from this I expect it to be an int, but below you're returning the original `game` instead of the one that's populated by this call

Reply:
Fixed, thanks for calling that out. I actually had an extra populateGame call that I didn't even need.